### DISCUSSION TITLE
Migrate to new HA installation

### DISCUSSION THREAD

**Discussion by jjnj023** [Blackymas/NSPanel_HA_Blueprint#586] [Q&A] [answered]
Im currently trying to install a new Home Assistant machine. The NSPanels are recognised in the devices page. However in ESPHome they aren't visible. This means I will be unable to update the software and let them connect to the new machine to obtain the blueprint.
Does anybody know I solution? I already tried a backup of ESPHome on the old machine and restore it on the new machine, but they don't appear.

**Comment by edwardtfn** [comment, marked as answer]
Have you tried copying the /config/esphome/*.yaml files from the old machine to the new one?
Even if the panel is not online this way, it should be quite straightforward to install the ESPHOME again in the panel and it should be connected to your ESPHOME dashboard after that.

**Comment by jjnj023** [comment]
A thanks, I'll try that!

**Comment by jjnj023** [comment]
I ended up using a full backup.
Thanks for your suggestion!
